.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
```suggestion
pnpm run lint-staged
```
wouldn't this be better? We have locally installed `lint-staged`

Reply:
It is not a script. At least for me with run it was not working. And lint-staged is not below scipts in packages.